Fluent/Post_Process.ipynb

```diff
@@ -6,127 +6,147 @@
     "\n",
-    "Use double `$$ <equation> $$` symbol to put equation in standalone format in the center of the page:\n",
+    "This pressure gradient gives rise to a downward (negative) lift on the cylinder. This was called the magnus effect produced by rotation of the cylinder.\n",
     "\n",
-    "$$\n",
-    "\\mathbf{R_{ext}} + \\mathbf{F_v} -\n",
-    "\\int \\mathbf{n}~p~dA +\n",
-    "\\int \\rho \\mathbf{g}~dV = \n",
-    "\\frac{\\partial}{\\partial t} \\int \\rho \\mathbf{V} dV +\n",
-    "\\int (\\mathbf{n}~.~\\rho \\mathbf{V}) \\mathbf{V} dA .\n",
-    "$$\n",
     "\n",
+    "<img src=\"./Images/streamlines.png\" width = \"600\">\n",
+    "Fig. 3. Streamlines for different $\\alpha$.\n",
     "\n",
-    "## Cite Source(s)\n",
-    "```\n",
-    "> For more details on the physics, theory and equation derivation please see chapter 9, section 9.2 of \"A Physical Introduction to Fluid Mechanics by Alexander J. Smits\" 2nd edition. [Download Book Here!](http://www.efluids.com/efluids/books/efluids_books.htm)\n",
-    "```\n",
+    "Unsteady flow over cylinder gives rise to alternating vortex shedding behind the cylinder. This vortex shedding has a shedding frequency. A non dimensionl number, Strouhal number (St) is associated with this frequency. Strouhal number is defined as $St = D\\omega/U_\\infty$, where D is the diameter of the cylinder. When $D = 1$ and $U_\\infty =1$, St is simply $\\omega$ as in this case. Strouhal number is atio of the unsteadiness in the flow to inertial forces in the flow field. Calculate the value of St for different vlaues of $\\alpha$ and compare with other experimental and numerical results. Here the values are compared with the results of Kang et al (2006)[3] & Mittal and Kumar (2003)[4] (shown to compare the trend of St with α). The results seem to be in good agreement with each other with an difference of 6.25%. The CFD results do not exactly match the experimental values. THe experimental and CFD setups do not match exactly. It is also not possible to exactly recreate the experimental environment in the software. This leads to the variation in the results. The expected trend for Strouhal number is that it remains almost constant for the given range of $\\alpha$. For $\\alpha = 2$, the CFD value is decreased. The reason for this is not determined yet. \n",
     "\n",
-    "> For more details on the physics, theory and equation derivation please see chapter 9, section 9.2 of \"A Physical Introduction to Fluid Mechanics by Alexander J. Smits\" 2nd edition. [Download Book Here!](http://www.efluids.com/efluids/books/efluids_books.htm)"
+    "<img src=\"./Images/strouhal2.png\" width = \"600\">\n",
```

Comment:
The content below this section was not reviewed anymore. Please modify the content base on the above reviews and comments and resubmit for review.